test/unittest/tests/test_acquire_release_macros.cpp

```diff
@@ -0,0 +1,80 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include <catch2/catch.hpp>
+
+#include "wt_internal.h"
+
+#define TEST_ACQUIRE_TYPE(type)                      \
+    TEST_CASE("Test acquiring a " #type, "[acqrel]") \
+    {                                                \
+        type a = 5;                                  \
+        type *ap = &a;                               \
+        type a_result;                               \
+        WT_ACQUIRE_READ(a_result, *ap);              \
+        REQUIRE(a == a_result);                      \
+                                                     \
+        type b;                                      \
+        type *bp = &b;                               \
+        type b_result = 5;                           \
```

Comment:
Is this test passing? We never set the value of `b`

Reply:
My guess is both were zero, this is C++ not C after all. ec9d34273